emcc.py

```diff
@@ -2410,6 +2410,7 @@ def generate_html(target, options, js_target, target_basename,
             } else if (Module['memoryInitializerPrefixURL']) {
               memoryInitializer = Module['memoryInitializerPrefixURL'] + memoryInitializer;
             }
+            if (Module['scriptDirectory'] && !/^(?:[a-z]+:)?\/\//i.test(memoryInitializer)) memoryInitializer = Module['scriptDirectory'] + memoryInitializer;
```

Comment:
a comment explaining this would be good i think

Reply:
check